lib/dor/models/publishable.rb

```diff
@@ -50,7 +50,6 @@ def public_xml
       rels = public_relationships.root
       pub.add_child(rels.clone) unless rels.nil? # TODO: Should never be nil in practice; working around an ActiveFedora quirk for testing
       pub.add_child(generate_dublin_core.root.clone)
-      @public_xml_doc = pub # save this for possible IIIF Presentation manifest
       pub.add_child(Nokogiri(generate_release_xml).root.clone) unless release_xml.children.size == 0 # If there are no release_tags, this prevents an empty <releaseData/> from being added
```

Comment:
The code for IIIF Presentation manifests was moved into the stacks server, so this is no longer needed and can be deleted.